libtiledbsoma/test/common.cc

```diff
@@ -33,11 +33,12 @@
 #include "common.h"
 
 namespace helper {
-ArraySchema create_schema(Context& ctx, bool allow_duplicates) {
+ArraySchema create_schema(
+    Context& ctx, int64_t dim_max, bool allow_duplicates) {
     // Create schema
     ArraySchema schema(ctx, TILEDB_SPARSE);
 
-    auto dim = Dimension::create<int64_t>(ctx, "d0", {0, 1000});
+    auto dim = Dimension::create<int64_t>(ctx, "d0", {0, dim_max});
```

Comment:
This is atop #2910, which should be approved/merged first, and then this distraction will rebase away.